src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -50,15 +50,49 @@ let uniqueId = 0;
 })
 export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDestroy {
 
+  /**
+   * Specifies the index of the repeater item to visually highlight as active.
+   * For example, use this property in conjunction with the
+   * [split view component](https://developer.blackbaud.com/skyux/components/split-view)
+   * to highlight a repeater item while users edit it. Only one item can be active at a time.
+   */
   @Input()
   public activeIndex: number;
 
+  /**
+   * Defines a string value to label the repeater list for accessibility.
+   * @default 'List of items'
+   */
   @Input()
   public ariaLabel: string;
 
+  /**
+   * Indicates whether users can change the order of items in the repeater list.
```

Comment:
Can reordering of items be done programmatically? If so, I assume we should reword this property description and the `orderChange` event description.

Reply:
Good question, but it looks like we don't offer any way through the public API to reorder items yet. I think this is ok as is for now.